packages/coturn/project.bri

```diff
@@ -0,0 +1,55 @@
+import * as std from "std";
+import { cmakeBuild } from "cmake";
+import libevent from "libevent";
+import openssl from "openssl";
+
+export const project = {
+  name: "coturn",
+  version: "4.7.0",
+  repository: "https://github.com/coturn/coturn.git",
+};
+
+const source = Brioche.gitCheckout({
+  repository: project.repository,
+  ref: project.version,
+});
+
+export default function coturn(): std.Recipe<std.Directory> {
+  return (
+    cmakeBuild({
+      source,
+      dependencies: [std.toolchain, libevent, openssl],
+    })
+      // Fix broken absolute-path symlink
+      .insert("bin/turnadmin", std.symlink({ target: "turnserver" }))
+      .pipe((recipe) =>
+        // Ensure all binaries are marked as executable
+        std.runBash`
+          find bin -type f -exec chmod +x {} +
+        `
```

Comment:
None of the binaries in the output were marked as executable from what I can tell. No clue why that is... so this is a fairly simple workaround